raster-foundry/raster-foundry
# Add Layer ID to Annotations and Annotation Groups

## Overview

This PR adds `project_layer_id` to `annotations` and `annotation_groups` tables. It also updates affected data models, `Dao`s, and property tests.

### Checklist

- ~Styleguide updated, if necessary~
- [x] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated (https://github.com/raster-foundry/raster-foundry-api-spec/pull/84)
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

## Testing Instructions

 * Run migration
 * CI
 * Use annotation frontend and make sure it works as expected (we have a known issue for annotation shapefile export: https://github.com/raster-foundry/raster-foundry/issues/4557)

Closes https://github.com/raster-foundry/raster-foundry/issues/4528
